scripts/generate-releases.sh

```diff
@@ -74,22 +75,28 @@ generate_release_resources() {
         snapshot_copy="${snapshot%-*}-${release_name}" # replace random suffix with release name
         echo "---"
         kubectl -n rh-acs-tenant get snapshot.appstudio.redhat.com "${snapshot}" -o yaml | \
-        yq -P 'load("whitelist.yaml") as $whitelisted
+        yq -P 'load("'"${whitelist_file}"'") as $whitelisted
          | del(.metadata.annotations |keys[]|select(. as $needle | $whitelisted.annotations | has($needle) | not))
          | del(.metadata.labels |keys[]|select(. as $needle | $whitelisted.labels | has($needle) | not))
          | del(.metadata.ownerReferences, .metadata.generation, .metadata.uid, .metadata.generateName, .metadata.creationTimestamp, .metadata.resourceVersion)
-         ' | sed "s,name: ${snapshot}$,name: ${snapshot_copy},"
+         | del(.status)
+         | .metadata.name = "'"${snapshot_copy}"'"
+         | .metadata.annotations["acs.redhat.com/original-snapshot-name"] = "'"${snapshot}"'"'
         application="$(echo "$line" | cut -d "|" -f 2)"
         release_plan="${application/acs-operator-index/acs-operator-index-${ENVIRONMENT}}"
-        echo "---
+
+        cat <<EOT
+---
 apiVersion: appstudio.redhat.com/v1alpha1
 kind: Release
 metadata:
   name: ${application}-${release_name_short}
   namespace: rh-acs-tenant
 spec:
   releasePlan: ${release_plan}
-  snapshot: ${snapshot_copy}"
+  snapshot: ${snapshot_copy}
+EOT
```

Comment:
Please try to find a way to have this indented. Maybe something like this will work?

```suggestion
        sed -E 's/^[[:blank:]]{8}//' <<<"
        ---
        apiVersion: appstudio.redhat.com/v1alpha1
        kind: Release
        metadata:
          name: ${application}-${release_name_short}
          namespace: rh-acs-tenant
        spec:
          releasePlan: ${release_plan}
          snapshot: ${snapshot_copy}"
```

If this does not feel right, I'd be ok if you wrap this unindented in a separate function.